Developers/Writing_Bears.rst

```diff
@@ -150,6 +150,42 @@ This will show the user a bunch of information related to the bear like:
 - A description of what the bear does - The sections which uses it - The
 settings it uses (optional and required)
 
+.. note::
+
+    The bears are not yet installed. We still have to specify
+    the bear directory using ``-d`` or ``--bear-dirs`` flag.
+
+
+Installing the New coala-bears Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install new coala-bear locally make sure :doc:`Bear Installation Tool
+(cib) <./Bear_Installation_Tool>` is installed in our environment.
+
+Let's say NewBear.py contain our NewBear. To install our NewBear :
+
+-  Move the ``NewBear.py`` to our clone of coala-bears in
+   ``coala-bear/bears/<some_directory>``.
+
+-  Uninstall all bears with:
+
+::
+
+    cib uninstall all
+
+-  Reinstall all bears from source with:
+
+::
+
+    pip install <path/to/coala-bears>
+
+Try Executing:
+::
+
+    coala --show-bears
+
+This shows a list of all installed bears.
```

Comment:
im confused all in all about this new paragraph because cib is experimental and its not fully usable yet, so i dont recommend anyone to use it unless forced..